### DISCUSSION TITLE
Understanding and modifying config files for different architecture implemenations

### DISCUSSION THREAD

**Discussion by shrinidhin** [explosion/spaCy#10506] [Help: Coding & Implementations] [answered]
Hi!I have been using spacy for an entity based sentiment analysis project and have been loving it!I am still exploring the guides and documentations along with my implementations and have received a lot of help from the spacy team on github here!Thank you!

I have been wondering how to write the config or rather what to include in the config file, If I were to use different model architectures for my custom trained spacy pipelines. Taking the [Healthsea](https://github.com/explosion/healthsea) project as an example, For the Entity recognition bit after training on simple tok2vec embedding, If I want to use a transformer architecture and evaluate the results, I am not able to understand what to include in the config. Though the [github repo](https://github.com/explosion/healthsea) has the config files available. Is it required for any project to try out different model architectures? How do I understand the different architectures to make the relevant changes in the config files?

**Comment by thomashacker** [comment, marked as answer]
Hello,
If you want to use transformer embeddings for your components you can use the [quickstart widget](https://spacy.io/usage/training#quickstart) in our docs to get the pipeline configuration you want and then use `spacy init config` (https://spacy.io/api/cli#init) to auto-fill all default values.
